app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -259,6 +259,7 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope() {
         if (tunInterface == null) {
             logcat(LogPriority.ERROR) { "Failed to establish the TUN interface" }
             deviceShieldPixels.vpnEstablishTunInterfaceError()
+            stopVpn(VpnStopReason.ERROR, false)
```

Comment:
We are actually stopping here: https://github.com/duckduckgo/android-netp/blob/feature/netp-develop/app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt#L376

Reply:
With this changes,we will doing double stopVpn which I am not sure if it will have any implication.

Reply:
argh, missed that, reverting the change.
Merged it before this comment came in